src/data/automation_i18n.ts

```diff
@@ -896,6 +902,39 @@ const tryDescribeCondition = (
     }
   }
 
+  const description = describeLegacyCondition(
+    condition as LegacyCondition,
+    hass,
+    entityRegistry
+  );
+
+  if (description) {
+    return description;
+  }
+
+  const conditionType = condition.condition;
+
+  const domain = getTriggerDomain(condition.condition);
+  const type = getTriggerObjectId(condition.condition);
```

Comment:
This looks like a copy/paste remnant
```suggestion
  const domain = getConditionDomain(condition.condition);
  const type = getConditionObjectId(condition.condition);
```